camp/mmcadams/YogiClimbers/src/main/java/frc/robot/RobotContainer.java

```diff
@@ -135,8 +135,8 @@ private void configureButtonBindings() {
    */
   public Command getAutonomousCommand() {
     TelescopingArmExtendHigh high = new TelescopingArmExtendHigh(armsSubsystem);
-    TelescopingArmExtendHigh retract = new TelescopingArmExtendHigh(armsSubsystem);
-    SequentialCommandGroup group = new SequentialCommandGroup(high, retract);
+    TelescopingArmsRetract retract = new TelescopingArmsRetract(armsSubsystem);
+    SequentialCommandGroup group = new SequentialCommandGroup(high.withTimeout(5.0), retract.withTimeout(6.0));
```

Comment:
could you add a comment about what these timeouts are for and how you arrived at these values?

Reply:
agree ... made a change